vc/authorization-credential-v1.jsonld

```diff
@@ -0,0 +1,36 @@
+{
+   "@context":{
+      "@version":1.1,
+      "@protected":true,
+      "name":"http://schema.org/name",
+      "ex":"https://example.org/examples#",
+      "xsd":"http://www.w3.org/2001/XMLSchema#",
+
+      "AuthorizationCredential":"ex:AuthorizationCredential",
+      "DIDDoc":{
+         "@id":"ex:DIDDoc",
+         "@context":{
+            "@version":1.1,
+            "@protected":true,
+            "id":"@id",
+            "type":"@type",
+            "doc":{
+               "@id":"ex:doc",
+               "@type":"@json"
+            }
+         }
+      },
+      "issuerDIDDoc":{
+         "@id":"ex:issuerDIDDoc",
+         "@type":"DIDDoc"
+      },
+      "rpDIDDoc":{
+         "@id":"ex:rpDIDDoc",
+         "@type":"DIDDoc"
+      },
+      "userDID":{
```

Comment:
we should also rename user.

Reply:
@troyronda this is just the user's DID

Reply:
@troyronda any specific reason to rename ? Do you see that the value of this attribute could be `non-did` ?

Reply:
my hesitation is not the DID aspect but the “user” term :)

Reply:
Oh :) any suggestions ?

Reply:
subject would be the more generic form.

Reply:
[Updated](https://github.com/trustbloc/context/compare/b65a651f9b159da815616be3a65de5816be439b6..2faf6e8a0a9d9c3347e05b2d8ebcd43e7c315660).